learn/getting_started/algolia_migration.md

```diff
@@ -0,0 +1,249 @@
+| attributesToRetrieve | displayedAttributes |
+| attributeForDistinct | distinctAttribute |
+| ranking | rankingRules |
+| Custom Ranking | Custom Ranking |
+| removeStopWords | stopWords |
+| synonyms | synonyms |
+| Sorting(using replicas) | sortableAttributes(no replicas required)  |
+| removeWordsIfNoResults | Meilisearch automatically supports it. However, it isn't customizable. |
+| disableTypoToleranceOnAttributes | Coming in Meilisearch v0.27 |
+| separatorsToIndex | Not Supported |
+| disablePrefixOnAttributes | Not Supported |
+| relevancyStrictness | Not Supported |
+| maxValuesPerFacet | Not Supported |
+| sortFacetValuesBy | Not Supported |
+| restrictHighlightAndSnippetArrays | Not Supported |
+
+## APIs
+
+This section compares Algolia and Meilisearch's respective API methods, using JavaScript for reference.
+
+| Topic | Algolia | Meilisearch |
+| --- | --- | --- |
+| Index Instantiation | `client.initIndex()`<br>Here, client is an Algolia instance. | `client.index()`<br>Here, client is a Meilisearch instance. |
+| Create Index | Users don’t need to create an index explicitly, the engine does it for you the first time you add an object or set settings. | The same applies for Meilisearch but users can also create an index explicitly using: `client.createIndex(string indexName)` |
+| Get All Indexes | `client.listIndices()` | `client.getIndexes()` |
+| Get Single Index | No method available | `client.getIndex(string indexName)` |
+| Delete Index | `index.delete()` | `client.deleteIndex(string indexName)` |
+| Get Index Settings | `index.getSettings()` | `index().getSettings()` |
+| Update Index Settings | `index.setSettings(object settings)` | `index().updateSettings(object settings)` |
+| Search Method | `index.search(string query, { searchParameters, requestOptions })` | `index.search(string query, object searchParameters)` |
+| Add Object | `index.saveObjects(array objects)` | `index.addDocuments(array objects)` |
+| Partial Update Object | `index.partialUpdateObjects(array objects)` | `index.updateDocuments(array objects)` |
+| Delete All Objects | `index.deleteObjects(array objectIDs)` | `index.deleteAllDocuments()` |
+| Delete One Object | `index.deleteObject(string objectID)` | `index.deleteDocument(string id)` |
+| Get All Objects | `index.getObjects(array objectIDs)` | `index.getDocuments(object params)` |
+| Get Single Object | `index.getObject(str objectID)` | `index.getDocument(string id)` |
+| Get API Keys | `client.listApiKeys()` | `client.getKeys()` |
+| Get API Key Info | `client.getApiKey(string apiKey)` | `client.getKey(string apiKey)` |
+| Create API Key | `client.addApiKey(array acl)` | `client.createKey(object configuration)` |
+| Update API Key | `client.updateApiKey(string apiKey, object configuration)` | `client.updateKey(string apiKey, object configuration)` |
+| Delete API Key | `client.deleteApiKey(string apiKey)` | `client.deleteKey(string apiKey)` |
+
+## Front-end components
```

Comment:
This section feels incomplete, maybe use a table here instead?

Reply:
Do you mean front-end components? There used to be a table here, but we removed it after deciding it would be too muchtrouble to keep up-to-date. decided it would be too much trouble to keep it up-to-date. Probably better to send people to Instant Meilisearch's repo/documentation anyway.